{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/settings.py

```diff
@@ -47,9 +47,13 @@ class ProjectDefault(Configuration):
         "django.contrib.sessions",
         "django.contrib.messages",
         "django.contrib.staticfiles",
+        "rest_framework",
+        "cors_headers",
+        "drf_spectacular",
     ]
 
     MIDDLEWARE = [
+        "corsheaders.middleware.CorsMiddleware",
```

Comment:
is this strictly related to API documentation feature?
I would suggest doing a separate PR

Reply:
This PR has evolved in its scope, hence the new title.